step-controller/step-controller-base-plugins/src/test/java/step/plugins/parametermanager/ParameterManagerTest.java

```diff
@@ -110,7 +110,7 @@ public void testPerf() throws ScriptException {
 		System.out.println("ms:"+(System.currentTimeMillis()-t1));
 		Assert.assertEquals(params.get("key1"),"value"+nIt);
 		
-		Assert.assertTrue((System.currentTimeMillis()-t1)<500);	
+		Assert.assertTrue((System.currentTimeMillis()-t1)<5000);

```

Comment:
The performance of the method getAllParameterValues is critical as it is executed for each execution start. Did the changes of this PR impact the performance of this method?

Reply:
@jeromecomte : due to the importance of the tests, I switch it to use a mongo collection and the central DB, and compare the performance between master and the branch which are identical. Note that I add to increase the threshold to 2 seconds for mongo.